cupy/cutensor.pyx

```diff
@@ -89,7 +115,7 @@ def create_mode(*mode):
     return Mode(integer_mode)
 
 
-def _auto_create_mode(array, mode):
+cdef inline Mode _auto_create_mode(ndarray array, mode):
```

Comment:
```suggestion
cdef inline Mode _auto_create_mode(ndarray array, mode) except -1:
```

Reply:
We don’t need to declare exception values for functions that return a Python object.
https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#error-return-values

Reply:
Yeah I know. I could be wrong, but I remember Cython is being weird in this case and would not propagate the exception properly. Let me check later (on my phone now).

Reply:
I was wrong, probably had something else in mind, sorry 😅